packages/api/endpoints/elasticsearch.js

```diff
@@ -163,17 +163,22 @@ async function changeIndex(req, res) {
     return res.boom.badRequest(`New index ${newIndex} does not exist.`);
   }
 
-  await esClient.indices.updateAliases({
-    body: {
-      actions: [
-        { remove: { index: currentIndex, alias: aliasName } },
-        { add: { index: newIndex, alias: aliasName } }
-      ]
-    }
-  }).then(() => {
+  try {
```

Comment:
Is this change fixing a linter error?

Reply:
Yeah, I'm pretty sure it was addressing this one: https://github.com/xjamundx/eslint-plugin-promise/blob/master/docs/rules/always-return.md